doc/connectivity/bluetooth/api/mesh/brg_cfg_srv.rst

```diff
@@ -3,6 +3,43 @@
 Bridge Configuration Server
 ###########################
 
+The Bridge Configuration Server model is a foundation model defined by the Bluetooth Mesh
+specification. It is an optional model, enabled with the
+:kconfig:option:`CONFIG_BT_MESH_BRG_CFG_SRV` configuration option.
+
+The Bridge Configuration Server model was introduced in the Bluetooth Mesh Protocol Specification
+version 1.1, and is used for supporting and configuring the Subnet Bridge feature.
+
+The Bridge Configuration Server model relies on :ref:`bluetooth_mesh_models_brg_cfg_cli` to
+configure it. The Bridge Configuration Server model only accepts messages encrypted with the node’s
+device key.
+
+If present, the Bridge Configuration Server model must be instantiated on the primary element.
+
+The Bridge Configuration Server model provides access to the following three states:
```

Comment:
```suggestion
The Bridge Configuration Server model provides access to the following three states:

* Subnet Bridge state
* Bridging Table state
* Bridging Table Size state
```

Reply:
Alternatively, the sections under can be replaced by a bullet list or a definition list :)
I don't have any strong preferences, so I will leave it up to you :)